RELEASING.adoc

```diff
@@ -1,80 +1,177 @@
 = Releasing Kiali
 
-This document gives the steps required to perform a release of Kiali. This document is only useful to Kiali release managers.
+This document covers the specifics of releasing Kiali, and is meant as a guide
+for Kiali release managers.
+
+The summary of the process is the following:
+
+* Create a new JIRA issue for the release
+* Make sure Getting Started docs are complete
+* Update Kiali UI version
+* Publish the API Docs
+* Update Makefile
+* Create the Release PR
+* Tag Git Repo
+* Notify Istio and Openshift Istio Teams for changes:
+  * Istio Helm Charts
+  * Istio Ansible Installer
+  * Openshift Istio Ansible Installer (istiooc)
```

Comment:
The asciidoc preview  is showing these three lines above are not being listed as sub-bullets, but rather on the same level as the "Notify Istio..." line. I don't know why - this looks correct with the two-space indentation. Keep an eye on this when we merge - not sure if the preview is just wrong or if something really is wrong with the asciidoc here.

Reply:
I think the syntax for asciidoc for listings is different from markdown. I'll check.

Reply:
In asciidoc a blank line is required before and after a list to separated it from other blocks.